go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -212,6 +215,9 @@ func ConvertPodFailureToError(status v1.PodStatus) (code, message string) {
 					c.LastTerminationState.Terminated.ExitCode,
 					c.LastTerminationState.Terminated.Reason,
 					c.LastTerminationState.Terminated.Message)
+			} else if c.State.Terminated != nil && strings.Contains(c.State.Terminated.Reason, OOMKilled) {
+				code = OOMKilled
+				message += fmt.Sprintf("\r\nContainer [%v] terminated due to being OOMKilled", c.ContainerID)
```

Comment:
it may help users if we can display the resource settings for their pod in the message too.

Reply:
we already show the resource requests and limits in the task details tab in the UI!